jsonschema-validation.xml

```diff
@@ -575,21 +559,23 @@
                     validates against the provided schema.
                     Note the property name that the schema is testing will always be a string.
                 </t>
+                <t>
+                    A missing keyword has the same behavior as an empty schema.
```

Comment:
I find the phrasing "A missing key word...", does not make me think that the missing key word in question is THIS key word (as in, the key word in where the phrase is being used).
I would much rather prefer something like "Omitting this key word has the same effect as...". I feel the intent is far clearer.

If it's just me that feels this way, please say.